### PR TITLE
Add tronador documentation

### PR DESCRIPTION
**Pull request by ShahidHussain-srti**

### CODE REVIEW

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-243-ae053ede`

File: content/sre/tronador/overview.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+Tronador comes with a CRD and a Tekton cluster task. The CRD, `EnvironmentProvisioner` , will be used for creating the environment within your cluster. The Tekton cluster task, `create-environment-provisioner` , will be used for automatically creating the `EnvironmentProvisioner` Custom Resource. The task depends on a [Tronador config file](./config_file.html) to be created in your repository, which will be used to create the CR. The `create-environment-provisioner` task can be used within your own Tekton pipeline, automating the process of creating and deploying the image of your application after changes are made to it. An example of a pipeline doing this would be:
+
+1. Create a new branch in your repository
+2. Create a config file file in the branch
```
Comment (rasheedamir) [inline content/sre/tronador/overview.md]: @LilShah the word file is duplicated here

File: content/sre/tronador/environment_provisioner.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# EnvironmentProvisioner
+
+```yaml
+apiVersion: tronador.stakater.com/v1alpha1
+kind: EnvironmentProvisioner
+metadata:
+  name: stakater-nordmart-promotion-pr-9
```
Comment (rasheedamir) [inline content/sre/tronador/environment_provisioner.md:7]: this name needs to be unique?

File: content/sre/tronador/config_file.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      tag: {{APPLICATION_IMAGE_TAG}}
+```
+
+::: v-pre
+Their values must also be `{{APPLICATION_IMAGE_NAME}}` and `{{APPLICATION_IMAGE_TAG}}` respectively. These keys will be replaced with values provided from the cluster task in your CI pipeline that creates images after each commit.
+:::
+
+### namespaceLabels
+
+(Optional Field) Labels mentioned here will be applied into the namespace. If they are removed from here, they will be removed from the namespace as well. Labels in the namespace that were never in this field will not be affected. A possible application of this field is compatibility with [Tenant Operator](../tenant-operator/overview.html).
```
Comment (rasheedamir) [inline content/sre/tronador/config_file.md:58]: > A possible application

bit confusing statement

Comment (ShahidHussain-srti) [reply content/sre/tronador/config_file.md:58]: "A potential use case of this field is to allow compatibility with Tenant Operator" ?

File: content/sre/tronador/environment_provisioner.md (hunk 2 above)
Comment (ShahidHussain-srti) [reply content/sre/tronador/environment_provisioner.md:7]: yes, otherwise it will update the same ep

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-243-cee1dfd3`

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-243-73c8df78`

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-243-e304019a`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.181`

**Pull request merged**